### DISCUSSION TITLE
".pdf" File Not Found during Batch Processing of Multiple Files

### DISCUSSION THREAD

**Discussion by JustinKhoh** [ocrmypdf/OCRmyPDF#1424] [General]
Hi, 

I've been trying to OCR multiple files in my directory and have inserted in the code below:
find . -printf '%p\n' -name '*.pdf' -exec ocrmypdf '{}' '{}' \;

But I am receiving this error:
File not found - '*.pdf'

Is there something else that I must enter into the code?

Thank you!

**Comment by jbarlow83** [comment]
You need the backslash at the end, like so:
```
find . -name '*.pdf' -exec ocrmypdf '{}' '{}'  \;
```

**Reply by JustinKhoh** [reply]
Hi James,

I inserted the backslash at the end as shown in the image below but the problem still persists
![SS1](https://github.com/user-attachments/assets/76f80ad5-9631-488f-915e-75d8e9480224)

and there exist pdf files inside the directory as shown below
![ss2](https://github.com/user-attachments/assets/390c766f-c9a5-4fa4-bf0b-17c28b60289f)

Please advice.

Thank you!

**Comment by JustinKhoh** [comment]
Hi,

I've found a solution to my problem:
FOR %i IN (*.pdf) DO ocrmypdf "%i" "%i"

The code first finds a file ending with .pdf in the specified directory. Afterwards, it would perform the OCR procedure on the specified file. The code will do this for all .pdf files in the directory.
